# --- T2-COPYRIGHT-BEGIN ---
# t2/package/*/cosmic-applets/hotfix-cargo.patch
# Copyright (C) 2025 The T2 SDE Project
# SPDX-License-Identifier: GPL-2.0 or patched project license
# --- T2-COPYRIGHT-END ---

--- cosmic-applets-epoch-1.0.0-beta.1/justfile.vanilla	2025-09-22 19:29:43.784158825 +0200
+++ cosmic-applets-epoch-1.0.0-beta.1/justfile	2025-09-22 19:32:44.737790699 +0200
@@ -6,7 +6,7 @@
 target := if debug == '1' { 'debug' } else { 'release' }
 vendor_args := if vendor == '1' { '--frozen --offline' } else { '' }
 debug_args := if debug == '1' { '' } else { '--release' }
-cargo_args := vendor_args + ' ' + debug_args
+cargo_args := ''
 
 targetdir := env('CARGO_TARGET_DIR', 'target')
 sharedir := rootdir + prefix + '/share'
@@ -72,4 +72,4 @@
 [private]
 vendor-extract:
     rm -rf vendor
-    tar pxf vendor.tar
\ No newline at end of file
+    tar pxf vendor.tar
--- cosmic-applets-epoch-1.0.0-beta.1/justfile.vanilla	2025-09-22 19:36:57.624673569 +0200
+++ cosmic-applets-epoch-1.0.0-beta.1/justfile	2025-09-22 19:37:04.791352768 +0200
@@ -28,7 +28,7 @@
     cargo build {{args}}
 
 # Compiles with release profile
-build-release *args: (build-debug '--release' args)
+build-release *args: (build-debug '--release' cargo_args)
 
 # Compile with a vendored tarball
 build-vendored *args: vendor-extract (build-release '--frozen --offline' args)
